setup.py

```diff
@@ -107,7 +107,8 @@
   install_requires = ['six>=1.11.0', 'subprocess32; python_version < "3"'],
   extras_require = {
       'colors': ['colorama>=0.3.9'],
-      'crypto': ['cryptography>=2.6'],
+      'crypto:python_version < "3"': ['cryptography>=2.6,<3.4'],
+      'crypto:python_version >= "3.0"': ['cryptography>=2.6'],
```

Comment:
the version specifiers elsewhere are "3", but here it's "3.0" – any significance to that? Or just an extra level of precision (comparing to major.minor, instead of just major)?

Reply:
oops, that was not supposed to be there -- I had to experiment to find the format that setuptools would accept, that's just a leftover from the testing